mainapp/templates/mainapp/request_list.html

```diff
@@ -32,7 +33,24 @@
             {% endif %}
           </td>
           <td>{{ req.requestee }}</td>
-          <td><a href="tel:{{ req.requestee_phone }}" >{{ req.requestee_phone }}</a></td> 
+          <td><a href="tel:{{ req.requestee_phone }}">{{ req.requestee_phone }}</a></td> 
+          <td>
+            <a href="tel:{{ req.additional_phone1 }}">
```

Comment:
Please add `if` condition here also as `additional_phone1` is not required

Reply:
Shouldn't be much of a problem. I just added if so that the commas won't be displayed if not needed. Still, I'll update for uniformity.